lib/3_add_up_numbers.php

```diff
@@ -7,9 +7,34 @@
 // Run `bin/check` from the command line to execute the automated tests.
 
 function addUpNumbers($arr) {
-  // Your code goes here.
+  $total = 0;
+  for ($i = 0; $i < count($arr); $i++) {
+    $total += $arr[$i];
+  }
+  return $total;
 }
 
 // Write your own "tests" below. Refer to the the examples from exercise #1.
 
+echo "\nTesting that the array [1, 2, 3] adds up to 6.\n";
+if (addUpNumbers(array(1, 2, 3)) == 6) {
+  echo "Success!";
+}
+else{
+  echo "Failed!";
+}
+echo "\n-------------------------------------------------------------------\n";
+
+echo "\nTesting that the array [3, 2] adds up to 5.\n";
+if (addUpNumbers(array(3, 2)) == 5)  {
```

Comment:
I don't see this test scenario being meaningfully different than the first. Other scenarios you might consider testing: Does your method work for floating point numbers? A combination of integers and floating point? Negative numbers and positive numbers?